### ISSUE TITLE
"Insufficient office space" tipbox lingers after cursor moves away from the "Hire Employee" button

### ISSUE THREAD

**Issue by cigarmemr** [bitburner-official/bitburner-src#895]
### Step to reproduce
1. Create a corporation and expand to a new division
2. Hire employees by clicking "Hire Employee" button until the office is jammed
3. Watch out the popup tipbox "Insufficient office space"

**Edit:** Stable steps of reproducing is revealed:
1. Hire employees by clicking "Hire Employee" button until the office is jammed
THEN
2. Quickly move away the mouse curser BEFORE the tipbox has fully shown.

### Result
The tipbox lingers forever even after the mouse cursor moves away from the button.
![image](https://github.com/bitburner-official/bitburner-src/assets/26317456/219a0e59-9724-4740-8ad8-1fef12c2edc2)
It will disappear when another tipbox is poped-up, however.

**Edit:**
No. Once the bugged tipbox is lingering, it will not disappear even if other tipboxes are shown. The only means to make it disappear is to point the cursor at the "Hire Employee" button again, or switch to another tab without the "Hire Employee" button.

~~**Note:** Based on limited observations, this issue may not be triggered every time the steps above are performed. More tests are required to obtain accurate conditions.~~

### Game version
Bitburner v2.5.0 (b87b8b4be)

**Comment by catloversg** [comment]
@cigarmemr I cannot reproduce this issue. Can you try it again to see if it stills happen?

**Comment by catloversg** [comment]
@cigarmemr I cannot reproduce this issue. Can you try it again to see if it still happens?

**Comment by d0sboots** [comment]
I've seen this happen before, although I haven't played Corp recently so I don't know if it's still present.

**Comment by catloversg** [comment]
I can reproduce it now. The trick is that we need to test it on an old Chromium version.

How to reproduce it:
Option 1:
- Check out v2.5.0. This version uses Electron v22.2.1 (Chromium v108).
- Run `npm run electron`.
- Test it in the Electron app.

Option 2:
- Check out the latest version.
- Build it.
- Load the game in an old version of Chrome (v108) or Electron v22.2.1.

I believe that this bug is caused by the external libraries (MUI/React), browser bugs, or a combination of them. When I checked bugs at https://github.com/mui/material-ui/issues, I found multiple reports about Tooltip. One of them is this report: https://github.com/mui/material-ui/issues/22785. I don't think that we can do anything here. This bug is not reproducible with our current Electron version (26.3.0), latest Chrome, and latest Firefox, so I think it's not a problem anymore.
